lighthouse-core/test/gather/computed/trace-of-tab-test.js

```diff
@@ -54,7 +54,7 @@ describe('Trace of Tab computed artifact:', () => {
   describe('finds correct FMP', () => {
     it('if there was a tracingStartedInPage after the frame\'s navStart', async () => {
       const trace = await TraceOfTab.compute_(lateTracingStartedTrace);
-      assert.equal(trace.startedInPageEvt.ts, 29343544280);
+      assert.equal(trace.tracingIds.ts, 29343544280);
```

Comment:
these tests are more interested in making sure the correct things were selected from the trace than the particular `startedInPageEvt` timestamps. If dropping `tracingIds.ts`, the replacement for `trace.startedInPageEvt.ts` could be the `pid`, `tid`, or `frameId` (or all of them) with no loss of functionality to the test